cli/internal/cmd/upgradeapply.go

```diff
@@ -291,7 +316,7 @@ type imageFetcher interface {
 // upgradeAttestConfigIfDiff checks if the locally configured measurements are different from the cluster's measurements.
 // If so the function will ask the user to confirm (if --yes is not set) and upgrade the measurements only.
 func (u *upgradeApplyCmd) upgradeAttestConfigIfDiff(cmd *cobra.Command, newConfig config.AttestationCfg, flags upgradeApplyFlags) error {
```

Comment:
I have found it helpful in the past to add a unittest for functions like this. Functions that specify the output of the CLI. I found it helpful because it (a) forced me to specify the expected output, (b) made it easy to iterate all possible outputs and (c) made it easy to change the underlying string construction. Since there is a diff printed now, there is some complexity in the output generated. Potentially a unittest for `diffAttestationCfg` is easier/enough.  
Judge for yourself if you want to add such a test.

Reply:
I think that specifically trivial output tests are rarely useful and cause more maintenance for writing the test. If the output was a json format that is part of our API contract that would be different of course.

Reply:
Not doing trivial output tests has led me to fixing a bunch of bugs in the past, hence my recommendation ^^.
Again, judge for yourself.